src/handler.py

```diff
@@ -77,10 +77,24 @@ def gem_handler(body: Dict[str, Any], env_vars):
     try:
         if is_rank_command(body):
             return handle_rank_command()
+        
+        if is_opt_out_command(body):
+            user_discord_id: str = _get_sender_discord_id(body)
+            return handle_opt_out(user_discord_id)    
+        
+        if is_opt_in_command(body):
+            user_discord_id: str = _get_sender_discord_id(body)
+            return handle_opt_in(user_discord_id)
 
         gems_message: GemsMessage = GemsMessage.from_slash_command(body)
         LOGGER.info(f"Gem message: {gems_message}")
 
+        if not is_receiver_available(gems_message.receiver_discord_id):
+            return slash_command_response(f"""
+                **:pleading_face: {gems_message.receiver_discord_id} has chosen solitude and is temporarily not receiving any gems. 
```

Comment:
❤️ love this message

Reply:
Should we mention that they'll be back in 30 days?
Also should this messae be private?

Reply:
can we create a const file with all the emojis that we use? feels like its getting to the point where we have multiple emojis being thrown around

Reply:
> Should we mention that they'll be back in 30 days? Also should this message be private?

The reason the return date isn't specified is because users have the option to opt back in at any time. Regarding 'private', did you mean sending this message as a direct message?


> can we create a const file with all the emojis that we use? feels like its getting to the point where we have multiple emojis being thrown around

Yes, we should create a const file for both emojis and messages.

Reply:
I changed this part to display the discord username instead of the ID.